docs/source/tutorial/basic.rst

```diff
@@ -559,6 +559,7 @@ These extensions perform the following tasks:
 
 :class:`~training.extensions.Evaluator`
    Evaluates the current model on the test dataset at the end of every epoch.
+   It automatically switches to the test mode (see :ref:`configuration` for details), and so we do not have to take care on functions that behave differently in training/test modes (e.g. :func:`~chainer.functions.dropout`, :class:`~chainer.links.BatchNormalization`).
```

Comment:
"take care" seems a bit unclear. Do you mean:

and so we do not have to take any special action for functions that behave differently